packages/fxa-settings/src/components/UnitRowWithAvatar/index.tsx

```diff
@@ -25,13 +30,13 @@ export const UnitRowWithAvatar = ({ avatarUrl }: UnitRowWithAvatarProps) => {
       </div>
       <div className="unit-row-actions">
         <div>
-          <a
+          <Link
             className="cta-neutral transition-standard"
             data-testid="unit-row-with-avatar-route"
-            href="#"
+            to={'/beta/settings/avatar/change' + queryParams}
```

Comment:
Same question around needing to pass `queryParams` down into the URL. Also, I guess there's not a way to just make this `/avatar/change` instead of spelling out `/beta/settings/avatar/change`? I guess a find and replace for `/beta/settings` to `/settings` when we move this out of beta won't be so bad.

FWIW this can also be:

```
to={`/beta/settings/avatar/change${queryParams}`}
```

Reply:
> instead of spelling out /beta/settings/avatar/change?

I couldn't really find a good way to do this that didn't seem excessively complicated. It is probably ok for now.